src/components/application_manager/test/CMakeLists.txt

```diff
@@ -40,10 +40,11 @@ if (BUILD_TESTS)
     ${COMPONENTS_DIR}/utils/include/
     ${COMPONENTS_DIR}/resumption/include/
     ${COMPONENTS_DIR}/utils/include/
-    ${COMPONENTS_DIR}/policy/include/
+    ${POLICY_PATH}/include/
     ${COMPONENTS_DIR}/media_manager/include/
     ${COMPONENTS_DIR}/security_manager/include/
-    ${COMPONENTS_DIR}/policy/test/include/
+    ${POLICY_PATH}/policy/test/include/
```

Comment:
@LuxoftAKutsan 
According to your `set` in CMakeLists.txt file as below:
```
+   set(POLICY_PATH ${COMPONENTS_DIR}/policy/policy_external/)
+   set(POLICY_PATH ${COMPONENTS_DIR}/policy/policy_regular/)
```
Here seems should be  ```${POLICY_PATH}/test/include/```, if I am wrong, correct me.